src/ray/core_worker/core_worker.cc

```diff
@@ -1173,12 +1173,19 @@ Status CoreWorker::SetResource(const std::string &resource_name, const double ca
 std::unordered_map<std::string, double> AddPlacementGroupConstraint(
     const std::unordered_map<std::string, double> &resources,
     PlacementGroupID placement_group_id, int64_t bundle_index) {
+  if (bundle_index < 0) {
+    RAY_CHECK(bundle_index == -1) << "Invalid bundle index " << bundle_index;
```

Comment:
RAY_CHECK(bundle_index >= -1) << "Invalid bundle index " << bundle_index;

Reply:
Not needed.